### DISCUSSION TITLE
After installing FBW, I´ve got no reverse Thrust

### DISCUSSION THREAD

**Discussion by FNuno1962** [flybywiresim/aircraft#3637] [Help] [answered]
![TCA in IDLE](https://user-images.githubusercontent.com/79177436/108573544-cd885d80-730c-11eb-866c-16f030733bd5.png)
![TCA in Reverse](https://user-images.githubusercontent.com/79177436/108573580-eb55c280-730c-11eb-90f7-bfae6ad22edc.png)

I have installed the FBW version of A32NX and the reverse thrust doesn´t work. I use the TCA Throttle Quadrant and the minimum that I´ve got in the simulatoe, when I put the TCA Thrust lever in reverse, is IDLE. When I use the stable version (052), everything is ok.
First picture: Thrustmaster TCA Quadrant in the IDLE position
Second picture: Thrustmaster TCA Quadrant in full reverse.
Thank your help.

**Comment by cadilhac** [comment]
Not exactly the same config here (Precision Flight Controls throttle quadrant), but I noticed this with the custom FBW:
When my physical throttle levers are idle, they are higher in the aircraft (the bottom of the levers are aligned with the 0, like in your first image).

**Reply by FNuno1962** [reply]
I see that you have a similar problem. I sujest you follow the link
[#3065](https://github.com/flybywiresim/a32nx/discussions/3065) 
recomended to me by @JJKHub 
Nice flights

**Reply by cadilhac** [reply]
Not sure, as there are no reversers part on my throttles...

**Comment by JJKHub** [comment, marked as answer]
Did you already see this?

https://github.com/flybywiresim/a32nx/discussions/3065

Hope it helps you.

**Reply by FNuno1962** [reply]
It helped me a lot. Thank you
